libs/color-sensor/color.ts

```diff
@@ -7,7 +7,7 @@ const enum ColorSensorMode {
     Color = 2,
     RefRaw = 3,
     RgbRaw = 4,
-    ColorCal = 5,
+    Calibration = 5,
```

Comment:
Would be preferable to avoid making this change, or leave the ColorCal marked as `//% hidden`; if anyone had typescript code that referenced this there code would not work with this change

```suggestion
    //% hidden
    ColorCal = 5,
    Calibration = 5,
```

Might need to double check if hidden is supported in the version used here on enums, I fixed a regression in that relatively recently but not sure if it is present in this version https://github.com/microsoft/pxt/pull/9487

Reply:
It seemed to me that I returned as it was before :)